docs/developer/soffits/minimal_soffit.md

```diff
@@ -0,0 +1,71 @@
+
+# Minimal Soffit
+
+A soffit's interaction with the portal is HTTP-based.  It is posible to write a
+soffit in any language or platform that can accept, process, and respond to a
+connection over HTTP.  At the present time, the creators of Soffit expect to
+develop soffits mostly with [Java][] and [Spring Boot][].
+
+## Minimal Soffit Setup Instructions Using Spring Boot
+
+1. Use the [Spring Initializer][] to create a new Spring Boot project with the
+   following settings:
+    * Gradle Project (recommended)
+    * Packaging=*War* (recommended)
+    * Dependencies=*Cache* (recommended) & *Web* (required)
+    * Additional dependencies you intend to use (optional -- you can add them
+      later)
+1. Add Soffit as a dependency to your project (see below)
```

Comment:
> It requires human maintenance; if you reorder or add one in the middle, you're obligated to re-do them all

True, https://github.com/Jasig/uPortal/pull/796 can a least partially automate this.

> It's confusing; do the 1, 2, 3, 4 appear in the rendered document as plain text? Or is it markdown syntax? (With 1., 1., 1... it's clearer that Markdown manages the numbering)

Its clearer that Markdown does the ordering, its less clear what the actual ordering is while viewing source.

> But since you asked, I also did a little googling to see what others thought...

I based styling thus far off off the [default remark-lint value](https://github.com/wooorm/remark-lint/blob/master/doc/rules.md#ordered-list-marker-value) which is to include order.